cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v2/transactions.rst

```diff
@@ -0,0 +1,36 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-transactions:
+
+=============================
+Transactions HTTP RESTful API
+=============================
+
+Some internals of the Transaction System are exposed via HTTP REST endpoints.
+
+Size of Invalid Transactions
+----------------------------
+
+To get the number of invalid transactions in the system, issue an HTTP GET request::
+
+  GET <base-url>/transactions/invalid/size
+
+The response is a JSON string::
+
+  { "size": <size> }
+
+
+Truncate Invalid Transactions by time
+-------------------------------------
+
+To truncate invalid transactions before a specific time, issue an HTTP POST request::
+
+  POST <base-url>/transactions/invalid/remove/until
+
+with the timestamp in milliseconds as a JSON string in the body::
+
+  { "time" : <timestamp-ms> }
```

Comment:
It might be a good idea to add a note here saying -
It it important to make sure that the data written during invalid transactions has been removed until the timestamp used above. Otherwise, those writes will become visible to all future transactions. See ``ref Operations section`` for more information.